src/components/Chart.tsx

```diff
@@ -0,0 +1,156 @@
+/** Load dependencies */
+import React from "react";
+import { Chart as ChartJS } from "chart.js";
+import "chartjs-plugin-datalabels";
+
+/** Load types */
+import { Metric } from "../types";
+
+/** Types */
+type Context =
+  | string
+  | CanvasRenderingContext2D
+  | HTMLCanvasElement
+  | ArrayLike<CanvasRenderingContext2D | HTMLCanvasElement>;
+
+type RefContext =
+  | string
+  | ((instance: HTMLCanvasElement | null) => void)
+  | React.RefObject<HTMLCanvasElement>
+  | null
+  | undefined;
+
+export type ChartTypes =
```

Comment:
Aren't these "magic" values already defined in the ChartJS.
Can't we just get them from the there instead of repeating the definitions.
I assume you want something like a subset of types what your app will support but the values should be aligned with the already defined type in the ChartJS lib